mutyper/cli.py

```diff
@@ -231,8 +231,6 @@ def targets(args):
 def spectra(args):
     """subroutine for spectra subcommand."""
 
-    # NOTE: vcf must be instantiated with gts012=False (the default) due to cyvcf2
-    # num_unknown property bug https://github.com/brentp/cyvcf2/issues/236
```

Comment:
I would vote for leaving this or setting the minimum version of cyvcf2 to be a version that includes the fix. but you can ignore if you disagree.

Reply:
Good call—I've added cyvcf2 version enforcement to setup.py